### DISCUSSION TITLE
API routes out folder pages

### DISCUSSION THREAD

**Discussion by Pedromdsn** [vercel/next.js#37011] [Ideas]
### Describe the feature you'd like to request

Hi,
You know when the project starts to scale it starts to have many folders in the pages folder,
this makes accessing the api folder difficult.

### Describe the solution you'd like

I would like to propose a config in next.config.js to change the directory of the api folder to the project root (root or src).

### Describe alternatives you've considered

Another alternative would be to let the developer define in next.config.js where he wants the folder.

**Comment by ChrisCates** [comment]
Bumping this.
Separation of concern is really important when your SLOC increases.
It's a pretty trivial fix. Allowing the `/pages/api` folder to also be declared under `/api`.

Is there any way to get the devs to pay attention to this? I can throw some $ at it. IDK if they would accept my pull request lol.
